resources/lang/pl/authorization.php

```diff
@@ -42,47 +53,72 @@
         'message' => [
             'send' => [
                 'channel' => [
-                    'no_access' => 'Nie posiadasz dostępu do tego kanału.',
+                    'no_access' => 'Dostęp do tego kanału jest wymagany.',
```

Comment:
Mimo wszystko myślę, że Nie posiadasz dostępu do tego kanału jest lepsze bo to nowe odbiega chyba trochę od kontekstu